zzzz_modular_occulus/code/game/machinery/dorms_elevator.dm

```diff
@@ -158,10 +201,17 @@
 	desc = "An button used to call the elevator down to the current floor."
 	var/cooldown = 24 SECONDS
 	var/cooldown_timer
+	var/obj/machinery/computer/cryopod/elevator_storage/lostFound
 
 /obj/machinery/button/remote/elevator_call_button/trigger()
+	for(var/obj/machinery/computer/cryopod/elevator_storage/storage in range(usr, 5))
```

Comment:
Ditto, what if someone wants to map this further than 5 tiles from the thingy?

Reply:
Tori, why would the button for an elevator be more than five tiles away?

Reply:
What if someone makes a crazy elevator design that necessitates it!

Reply:
It won't be more than 5 tiles away, it's right next to the elevator...... if they want to map it to be further away, they can change the damn range themselves lmao